react/src/components/TimeContainer.tsx

```diff
@@ -1,22 +1,59 @@
+import { useBaiSignedRequestWithPromise } from '../helper';
+import { useUpdatableState } from '../hooks';
+import { useTanQuery } from '../hooks/reactQueryAlias';
+import BAIIntervalText from './BAIIntervalText';
+import Flex from './Flex';
+import { ClockCircleOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
 import { default as dayjs } from 'dayjs';
-import React, { useState, useEffect } from 'react';
+import React, { useTransition } from 'react';
+import { useTranslation } from 'react-i18next';
 
-interface TimeContainerProps {}
+interface TimeContainerProps {
+  format?: string;
+}
 
-const TimeContainer: React.FC<TimeContainerProps> = () => {
-  const [date, setDate] = useState(new Date());
+const TimeContainer: React.FC<TimeContainerProps> = ({
```

Comment:
It's better to use a more explicit name. `TimeContainer` => `LoginSessionExtendButton`